azkaban-common/src/main/java/azkaban/executor/ExecutableFlow.java

```diff
@@ -259,7 +278,15 @@ public void fillExecutableFromMapObject(
     if (flowObj.containsKey(PROXYUSERS_PARAM)) {
       List<String> proxyUserList = flowObj.<String> getList(PROXYUSERS_PARAM);
       this.addAllProxyUsers(proxyUserList);
+    }
 
+    if (flowObj.containsKey(SLAOPTIONS_PARAM)) {
+      List<Map<String, Object>> slaOptionsObject = flowObj.getList(SLAOPTIONS_PARAM);
+      List<SlaOption> slaOptions = new ArrayList<>();
+      for(Map<String, Object> slaOption : slaOptionsObject) {
```

Comment:
Why not use for(Map<String, Object> slaOption : flowObj.getList(SLAOPTIONS_PARAM))